packages/engine/src/physics/systems/PhysicsSystem.ts

```diff
@@ -134,8 +150,31 @@ export default async function PhysicsSystem(world: World) {
 
     // step physics world
     const substeps = engineState.physicsSubsteps.value
-    world.physicsWorld.timestep = engineState.fixedDeltaSeconds.value / substeps
+    const timestep = engineState.fixedDeltaSeconds.value / substeps
+    world.physicsWorld.timestep = timestep
+    const timestepMultiplier = 50
+    /*const maxDistance = 0.5
+    const maxDistanceSq = maxDistance * maxDistance*/
+    const kinematicBodies = [
+      ...kinematicPositionBodyQuery(),
+      ...kinematicVelocityBodyQuery()
+    ] /*.filter(kinematicBodyEntity => {
+      const transformComponent = getComponent(kinematicBodyEntity, TransformComponent)
```

Comment:
Let's just delete this commented out code, I don't think we'll need it. We can just add a tag to opt in or out of kinematic smoothing later.